pkg/cel/policies/vpol/compiler/policy.go

```diff
@@ -142,6 +142,10 @@ func (p *Policy) evaluateWithData(
 					message = msg
 				}
 			}
+			// Add default message if empty
+			if message == "" {
+				message = fmt.Sprintf("CEL validation at index %d failed", index)
```

Comment:
Can we modify the message to be `CEL expression validation failed at index %d` or something similar?